### PR TITLE
add brain_t1 category into exclude.yml file (Issue #126)

### LINKED ISSUE #126: Add "brain_t1" category and list of problematic scans into exclude.yml file
# FILE TO BE EXTENDED:
 [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml)

# BACKGROUND and CONVERSATION POINTING THE ISSUE

*"Remaining 18 T1w scans (6.74%) demonstrated strong cerebral field of view cut offs disabling proper cerebral image analysis. Thus, these scans were rather excluded from the analysis."*

**René Labounek • 7:35 AM, Jul 15 (CDT):** Make a new category in the  [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml) file?

**Julien Cohen-Adad • 8:31 AM, Jul 15 (CDT):** no need for another category, a simple comment will do-- see example:
[https://github.com/spine-generic/data-multi-subject/blob/082dd0d7b5ad1eedf53ea9765910e2d68769d355/exclude.yml#L49](https://www.google.com/url?q=https://github.com/spine-generic/data-multi-subject/blob/082dd0d7b5ad1eedf53ea9765910e2d68769d355/exclude.yml%23L49&sa=D&source=docs&ust=1657895489342976&usg=AOvVaw2A1KxkdWzyDQRtLJsJVqjp)

**Rene Labounek • 8:42 AM, Jul 15 (CDT):** I still think "brain_t1" category is needed, because csa_t1 lists you subjects where SC CSA measurement is impossible for some reasons (written as comment). But there are scans where you can measure SC CSA but not measure all cerebral volumes due to FOV cutting half of the brain. brain_t1 would list problematic subjects and comment can clarify the exact reason.

**Julien Cohen-Adad • 9:02 AM, Jul 15 (CDT):** ah! yes you are correct, i thought that 't1' was a category for the T1w scan, but it is not. Yes, you made a good point. Can you please open an issue and move the conversation we are currently having in the issue, so it is documented (and a subsequent PR will refer to that issue). thx!

# TASKS

@mbondy023, please, can you send us list of the 18 problematic scans and descriptive reason why FS image analysis is imposible for these particular scans? + Are there any other scans where FS analysis will be imposible after the full done re-defacing? If you would keep text formating as in the [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml) file, it would be the best.

Then, @renelabounek or @mbondy023 can make a new branch, edit [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml), submit PR and @jcohenadad or other admin accept the PR.

# PREVIOUS ISSUE REPORTING BRAIN FOV PROBLEM
#115

### PR DESCRIPTION
**Pull request by mbondy023**
Regarding issue #126 I have created the brain_t1 category in the [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml) file. There are **22** scans marked as unusable by Freesurfer. 

**Wrong FOV = 18 scans**
Example below
Screen Shot 2022-07-18 at 3 05 54 PM

**Signal loss = 4 scans**
Example below
Screen Shot 2022-07-18 at 3 32 31 PM

### CODE REVIEW

**Review by mguaypaq — APPROVED** [review]
This looks fine, as described in the linked issue, and BIDS validation is passing, so I'm approving this pull request.
